Rakefile

```diff
@@ -151,18 +146,13 @@ task :publish_aws_analytics do
   stop_chef_zero
 end
 
-########################
-#
 # Google
-#
-
 desc 'Publish GCE Marketplace Images'
 task :publish_gce, :marketplace, :product do |_, params|
   begin
     write_client_json('marketplace_image' => params.to_h, 'gce' => gce_config)
     start_chef_zero
-    berks_install
-    hosted_berks_install
+    berks_install(true)
```

Comment:
Still need a `berks_install` or `berks_install(false)` to populate the local chef-zero.